run_pipeline.py

```diff
@@ -1,63 +1,107 @@
 import logging
 import os
-from config import s3_endpoint_url, s3_bucket, s3_folder_in_bucket
+import time
+from typing import Optional
+from config import (
+    data_base_dir,
+    s3_endpoint_url,
+    ae_samplerate_hz,
```

Comment:
Not a must, but we could consider passing things like sample rate as body to the APIrequest, rather than obtain them from the config. The service would be more flexible that way.